Marlin/Marlin_main.cpp

```diff
@@ -4945,7 +4944,7 @@ inline void gcode_M226() {
   }
 #endif
  
-#ifdef E_REGULATOR
+#if ENABLED(E_REGULATOR)
   /**
    * M236 - Send Value to ADC w/ no EEPROM write
    */
```

Comment:
I have another branch to take care of that. `E_REGULATOR` will not be defined unless `PNEUMATICS` is defined.